spec/features/project/show_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe "the project show page" do
+  before(:each) do
+    @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    
+    @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+
+    ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+  end
+
+  it "shows details about the project and the theme of associated challenge" do
+  # As a visitor,
+
+    visit("/projects/#{@news_chic.id}")
+  # When I visit a project's show page ("/projects/:id"),
+  # I see that project's name and material
+    expect(page).to have_content("News Chic")
+    expect(page).to have_content("Newspaper")
+  # And I also see the theme of the challenge that this project belongs to.
+  # (e.g.    Litfit
+  #     Material: Lamp Shade
+  #   Challenge Theme: Apartment Furnishings)
+    expect(page).to have_content("Recycled Material")
+  end
+
+  it "shows how many contestants on each project" do
+
+    #     As a visitor,
+    # When I visit a project's show page
+    visit("/projects/#{@news_chic.id}")
+    # I see a count of the number of contestants on this project
+    expect(page).to have_content("2")
```

Comment:
This assertion could be more robust by anchoring it to a text label. For example `expect(page).to have_content("Number of Contestants: 2")`

that way, we're sure the 2 is showing up in the right place (what if there was a 2 in the name of the challenge? This would give a false positive in the current test set up)